src/main/java/seedu/duke/Duke.java

```diff
@@ -0,0 +1,76 @@
+package seedu.duke;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Timer;
+import java.util.TimerTask;
+
+import javafx.application.Platform;
+
+/**
+ * Main class to run Duke program.
+ * Creates and stores tasks such as todo, event and deadline.
+ */
+
```

Comment:
Perhaps this newline can be omitted? The style guide [suggests](https://se-education.org/guides/conventions/java/index.html#comments) a certain form for Javadocs, doesn't seem to have a line break between.